pkg/network/usm/sharedlibraries/compile.go

```diff
@@ -18,7 +18,7 @@ import (
 //go:generate $GOPATH/bin/integrity pkg/ebpf/bytecode/build/runtime/shared-libraries.c pkg/ebpf/bytecode/runtime/shared-libraries.go runtime
 
 func getRuntimeCompiledSharedLibraries(config *config.Config) (runtime.CompiledOutput, error) {
-	return runtime.SharedLibraries.Compile(&config.Config, getCFlags(config), statsd.Client)
+	return runtime.SharedLibraries.Compile(&config.Config, getCFlags(config), getLlcFlags(), statsd.Client)
```

Comment:
same comment as above

Reply:
Same as above, this function will return valid `llc` flags in subsequent PRs.